src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const keysValues = sourceString.split(';');
+
+  const cssObject = {};
+
+  keysValues.forEach((keyValue) => {
+    const trimmedKeyValue = keyValue.trim();
+
+    if (trimmedKeyValue) {
+      const parts = trimmedKeyValue.split(':');
```

Comment:
When splitting 'keyValue' by ':', consider handling cases where the property value might contain additional colons, such as URLs or content with escaped characters. You can use 'indexOf' or 'slice' to handle such cases.